components/resourceDetails/resourceSharing/agentAccessList/index.test.jsx

```diff
@@ -56,24 +50,24 @@ describe("AgentAccessList", () => {
   it("renders a about empty list of permissions", () => {
     expect(
       mountToJson(
-        <SessionProvider>
+        <AccessControlProvider accessControl={accessControl}>
           <DatasetProvider dataset={dataset}>
             <AgentAccessList />
           </DatasetProvider>
-        </SessionProvider>
+        </AccessControlProvider>
       )
     ).toMatchSnapshot();
   });
 
   it("renders a list of permissions", () => {
-    permissionHelpers.getPermissions.mockResolvedValue([permission]);
+    accessControl.getPermissions = () => Promise.resolve([permission]);
```

Comment:
can we do this without mutating the shared `accessControl`? Works fine for now, but if we add subsequent tests then they'll be affected by the order tests run in

Reply:
Good point, I've changed this so that each test have their own unique instance of a mocked access control.